src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -310,7 +310,10 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
         }
       })
     } else {
-      mediator.trigger("open:auth", { mode: "login" })
+      mediator.trigger("open:auth", {
+        mode: "login",
+        redirectTo: location.href,
```

Comment:
Does the login flow not redirect you back to where you were when you started (which can then be overridden by passing in a `redirectTo`)?